src/components/DampAndMouldReports/components/DampAndMouldReportsPropertyReportsView.js

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from 'react'
+import { fetchReports } from '../fetchReports'
+import ErrorMessage from '../../Errors/ErrorMessage'
+import Spinner from '../../Spinner'
+import DampAndMouldReportTable from './DampAndMouldReportTable'
+import { Button } from '../../Form'
+
+const DampAndMouldReportsPropertyReportsView = ({ propertyReference }) => {
+  const pageSize = 20
+  const [pageNumber, setPageNumber] = useState(1)
+
+  const [reportsApiResponse, setReportsApiResponse] = useState(null)
+  const [reports, setReports] = useState([])
+
+  const [error, setError] = useState(null)
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    setIsLoading(true)
+
+    fetchReports(pageNumber, pageSize, propertyReference)
+      .then((reportsResponse) => {
+        setReportsApiResponse(reportsResponse)
+
+        setReports((existingReports) => [
+          ...existingReports,
+          ...reportsResponse.results,
+        ])
+      })
+      .catch((err) => {
+        console.error(err)
+        setError(err.message)
+      })
+      .finally(() => {
+        setIsLoading(false)
+      })
+  }, [])
+
+  const handleShowMoreResults = () => {
+    setIsLoading(true)
+
+    const newPageNumner = pageNumber + 1
+
+    setPageNumber((x) => x + 1)
+
+    fetchReports(newPageNumner, pageSize, propertyReference)
+      .then((res) => {
+        setReports((existingReports) => [...existingReports, ...res.results])
+      })
+      .catch((err) => {
+        console.error(err)
+        setError(err.message)
+      })
+      .finally(() => {
+        setIsLoading(false)
+      })
+  }
+
+  const isMorePages = reportsApiResponse?.pageCount > pageNumber ?? false
+
+  return (
+    <div className="govuk-body">
+      {error && <ErrorMessage label={error} />}
+
+      <p style={{ marginTop: '30px' }}>
+        Showing {reports.length ?? 0} of {reportsApiResponse?.totalCount ?? 0}{' '}
+        reports
+      </p>
+      {isLoading ? (
+        <Spinner />
+      ) : (
+        <>
+          <DampAndMouldReportTable
+            reports={reports}
+            showOtherReportsColumn={false}
```

Comment:
You could consider using a single boolean for both column flags to make the intent more clear, e.g. `showAdditionalPropertyInfo`.

Reply:
Yeah, if you think its clearer

Reply:
It took me a bit to figure out why these attributes existed